app/views/site/_study_download_data.html.erb

```diff
@@ -122,6 +122,8 @@
               var authCode = response.authCode;
               var timeInterval = response.timeInterval; // token expires after this many seconds
               var expiresMinutes = timeInterval / 60;
+              const isWindows = !!window.navigator.platform.match(/Win/) // determine client OS for formatting
+              const curlExec = isWindows ? 'curl.exe' : 'curl'
```

Comment:
some duplicated javascript between this and getDownloadCommand().  Tricky to refactor out though, since they're in the rails portion of the app, so you can't just import from a lib function